site/features.md

```diff
@@ -3,622 +3,262 @@ title: Weave Features
-    PING 10.2.1.2 (10.2.1.2): 48 data bytes
-    --- 10.2.1.2 ping statistics ---
-    1 packets transmitted, 1 packets received, 0% packet loss
-    round-trip min/avg/max/stddev = 1.048/1.048/1.048/0.000 ms
 
-Similarly, in the container on $HOST2...
+###<a name="plugin"></a>Weave Network Docker Plugin
 
-    root@04c4831fafd3:/# ping -c 1 -q 10.2.1.1
-    PING 10.2.1.1 (10.2.1.1): 48 data bytes
-    --- 10.2.1.1 ping statistics ---
-    1 packets transmitted, 1 packets received, 0% packet loss
-    round-trip min/avg/max/stddev = 1.034/1.034/1.034/0.000 ms
+Weave can also be used as a Docker plugin.  A Docker network 
+named `weave` is created by `weave launch`, which is used as follows:
 
-The IP addresses and netmasks can be anything you like, but make sure
-they don't conflict with any IP ranges in use on the hosts or
-IP addresses of external services the hosts or containers need to
-connect to. The individual IP addresses given to containers must, of
-course, be unique - if you pick an address that the automatic
-allocator has already assigned you will receive a warning.
+    $ docker run --net=weave -ti ubuntu 
 
-If you restart a container, it will retain the same IP addresses on
-the weave network:
+Weave’s Docker Network plugin doesn't require an external 
+cluster store. You can also start and stop containers even 
+when there are network connectivity problems.
 
-    host1$ docker run --name a1 -tdi ubuntu
-    f76b09a9fcfee04551dbb8d951d9a83e7e7d55126b02fd9f44f9f8a5f07d7c96
-    host1$ weave ps a1
-    a1 1e:dc:2a:db:ef:ff 10.32.0.3/12
-    host1$ docker restart a1
-    host1$ weave ps a1
-    a1 16:c0:6f:5d:c5:73 10.32.0.3/12
+>*Note:* The plugin is an *alternative* to the proxy, and therefor you do
+*not* need to run `eval $(weave env)` beforehand.
 
-It works the same if you stop and immediately restart:
+See [Using the Weave Docker Network Plugin](/site/plugin/weave-plugin-how-to.md) for more details.
 
-    host1$ docker stop a1
-    host1$ docker start a1
 
-However, additional addresses added with the `weave attach` command
-will not be retained.
+###<a name="addressing"></a>Address Allocation (IPAM)
+ 
+Containers are automatically allocated a unique IP address 
+across the weave network. Addresses allocated by Weave can 
+be viewed by running `weave ps`.
 
-There is also a `weave restart` command, which does re-attach all
-current IP addresses:
+Instead of allowing Weave to automatically allocate 
+addresses, an IP address and a network can be explicitly 
+specified using [CIDR notation](http://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation). 
+See [How to Manually Specify IP Addresses and Subnets(/site/using-weave/manual-ip-address.md)
 
-    host1$ weave restart b1
+For a discussion on how Weave uses IPAM, see 
+[Automatic IP Address Management](/site/ipam/overview-init-ipam.md) and also 
+[the basics of IP addressing](/site/ip-addresses/configuring-weave.md) 
```

Comment:
Think this link should be `ip-addresses.html`; certainly it's not about configuring weave.

Maybe this link could go earlier in the section, since it's aimed at newcomers.